### ISSUE TITLE
Add hack

### ISSUE THREAD

**Issue by rdbende** [rdbende/Sun-Valley-ttk-theme#97]
From Akuli/porcupine:
```py
# TODO: the next 3 lines are a hack?
# If they are really needed, why it isn't in sv-ttk by default?
main_window.option_add("*Text.highlightThickness", "0")
main_window.option_add("*Text.borderWidth", "2")
main_window.option_add("*Text.relief", "solid")
```

**Comment by littlewhitecloud** [comment]
Add hack means add them to the sv_ttk as default?

**Comment by rdbende** [comment]
Yeah, though I'm not sure, if it makes sense to make them the default.

**Comment by littlewhitecloud** [comment]
I agree with the first two, but I prefer flat to solid.

**Comment by littlewhitecloud** [comment]
Hightlightthickness is 0 means the Text widget will not show the highlight border and I agree this, because the border looks ugly.
![image](https://github.com/rdbende/Sun-Valley-ttk-theme/assets/71159641/b538d4dd-4c00-4b64-9807-3f6cf1c770ed)
And here is a screenshot that shows the difference between borderwidth=2 and the normal borderwidth, there are 1px black frame in the left and 2px grey frame in the bottom, so I disagree the second.
![image](https://github.com/rdbende/Sun-Valley-ttk-theme/assets/71159641/beb5434f-28c1-4a5f-b579-62cf151fe37f)
And the relief, here is the screenshot (test with borderwidth=2):
![image](https://github.com/rdbende/Sun-Valley-ttk-theme/assets/71159641/81754756-62bd-4def-aa22-47fd12cb637a)
I think normal relief is better than the solid, so I think they can set the solid by themself.
